services/community-site/src/services/useExplorerStats.tsx

```diff
@@ -34,8 +33,9 @@ export default () => {
           const diffHours = diff / 1000 / 60 / 60;
           return {
             ...validator,
+            pbftsProduced: stats.response.pbftCount,
```

Comment:
This is the total pbft blocks produced. We need the number of blocks produced this week